### PR TITLE
Add config_drive to Compute Server Struct

### PR DESCRIPTION
**Pull request by shimisho**
Added config_drive to the structure that stores the Get results of the compute server.

### CODE REVIEW

**Review by shisoAqron — APPROVED** [review]

File: ecl/compute/v2/servers/results.go [hunk 1]
```diff
@@ -214,6 +214,9 @@ type Server struct {
 
 	// Fault contains failure information about a server.
 	Fault Fault `json:"fault"`
+
+	// ConfigDrive is the name of the server's config drive.
+	ConfigDrive string `json:"config_drive"`
```
Comment (kazyshr) [inline ecl/compute/v2/servers/results.go:159]: API Doc shows that type of config_drive is boolean/UUID(2). Seems like it is inconsistent.
https://ecl.ntt.com/en/documents/api-references/virtual-server/rsts/APIList/Virtual_Server_Compute/Compute/Create_server.html#request-parameter

**Review by kazyshr — CHANGES_REQUESTED** [review]
Please look at my inline comment. And can you update tests?

File: ecl/compute/v2/servers/results.go (hunk 1 above)
Comment (shimisho) [reply ecl/compute/v2/servers/results.go:159]: This structure stores the result of Get API, so I think we should refer to the API Doc below.
According to this API Doc, the data type of config_drive is String.
https://ecl.ntt.com/en/documents/api-references/virtual-server/rsts/APIList/Virtual_Server_Compute/Compute/Get_server_details.html#response-parameter

**Comment by shimisho** [comment]
@kazyshr Thanks for your comment.Please check my reply.
The test did not exist, so I will create a new one.

File: ecl/compute/v2/servers/results.go (hunk 1 above)
Comment (kazyshr) [reply ecl/compute/v2/servers/results.go:159]: Thanks. I resolved.

**Comment by shimisho** [comment]
The following corrections have been made.

- Delete functions that cannot execute API
- Update Doc
- Added Compute Server MockTest

**Review by kazyshr — APPROVED** [review]
LGTM

**Pull request merged**